test/io/testmoldenwriter.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Unit tests for writer moldenwriter module."""
+
+import os
+import unittest
+
+import cclib
+
+
+__filedir__ = os.path.dirname(__file__)
+__filepath__ = os.path.realpath(__filedir__)
+__datadir__ = os.path.join(__filepath__, "..", "..")
+
+
+class MOLDENTest(unittest.TestCase):
+
+    def setUp(self):
+        self.molden = cclib.io.MOLDEN
+        self.MissingAttributeError = cclib.io.filewriter.MissingAttributeError
+
+    def test_init(self):
```

Comment:
MoldenWriter does not have it's own initializer, so do you need to test this again?